robonomics/cli/src/source.rs

```diff
@@ -64,6 +65,16 @@ pub enum SourceCmd {
         #[structopt(long, value_name = "HEARTBEAT_SECS", default_value = "5")]
         hearbeat: u64,
     },
+    /// Reading datalog.
+    Datalog {
+        /// Robonomics node API endpoint.
+        #[structopt(long, value_name = "REMOTE_URI", default_value = "ws://127.0.0.1:9944")]
+        remote: String,
+        /// Reader account seed URI.
+        #[structopt(short, value_name = "SECRET_URI")]
```

Comment:
Secret uri isn’t a good name for address, please change it to ADDRESS with key -a